homeassistant/generated/zeroconf.py

```diff
@@ -4,65 +4,242 @@
 """
 
 HOMEKIT = {
-    "3810X": "roku",
-    "3820X": "roku",
-    "4660X": "roku",
-    "7820X": "roku",
-    "819LMB": "myq",
-    "AC02": "tado",
-    "Abode": "abode",
-    "BSB002": "hue",
-    "C105X": "roku",
-    "C135X": "roku",
-    "EB-*": "ecobee",
-    "Escea": "escea",
-    "HHKBridge*": "hive",
-    "Healty Home Coach": "netatmo",
-    "Iota": "abode",
-    "LIFX A19": "lifx",
-    "LIFX BR30": "lifx",
-    "LIFX Beam": "lifx",
-    "LIFX Candle": "lifx",
-    "LIFX Clean": "lifx",
-    "LIFX Color": "lifx",
-    "LIFX DLCOL": "lifx",
-    "LIFX DLWW": "lifx",
-    "LIFX Dlight": "lifx",
-    "LIFX Downlight": "lifx",
-    "LIFX Filament": "lifx",
-    "LIFX GU10": "lifx",
-    "LIFX Lightstrip": "lifx",
-    "LIFX Mini": "lifx",
-    "LIFX Nightvision": "lifx",
-    "LIFX Pls": "lifx",
-    "LIFX Plus": "lifx",
-    "LIFX Tile": "lifx",
-    "LIFX White": "lifx",
-    "LIFX Z": "lifx",
-    "MYQ": "myq",
-    "NL29": "nanoleaf",
-    "NL42": "nanoleaf",
-    "NL47": "nanoleaf",
-    "NL48": "nanoleaf",
-    "NL52": "nanoleaf",
-    "NL59": "nanoleaf",
-    "Netatmo Relay": "netatmo",
-    "PowerView": "hunterdouglas_powerview",
-    "Presence": "netatmo",
-    "Rachio": "rachio",
-    "SPK5": "rainmachine",
-    "Sensibo": "sensibo",
-    "Smart Bridge": "lutron_caseta",
-    "Socket": "wemo",
-    "TRADFRI": "tradfri",
-    "Touch HD": "rainmachine",
-    "Welcome": "netatmo",
-    "Wemo": "wemo",
-    "YL*": "yeelight",
-    "ecobee*": "ecobee",
-    "iSmartGate": "gogogate2",
-    "iZone": "izone",
-    "tado": "tado",
+    "3810X": {
+        "domain": "roku",
+        "iot_class": "local_polling",
```

Comment:
can we pre-process this and store the result of the check on iot_class here??

Reply:
I adjusted it to build the dataclasses in `async_get_homekit` instead

Reply:
I completely misread your comment. Will revert what I did and refactor it now to be a flag that precalcs `not homekit_model.iot_class or (
                    not homekit_model.iot_class.startswith("cloud")
                    and "polling" not in homekit_model.iot_class
                )`